modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2201,6 +2202,9 @@ def apply_full_axis(
             The data types of the result. This is an optimization
             because there are functions that always result in a particular data
             type, and allows us to avoid (re)computing it.
+        keep_partitioning : boolean, default: True
```

Comment:
Why is this True by default? Shouldn't we make it False since `partition_mgr.broadcast_axis_partitions`, which is called further, has `keep_partitioning` equal to False by default?

Reply:
Answered in https://github.com/modin-project/modin/pull/5366/files#r1044417856

Reply:
Should we change the default value to True for `partition_mgr.broadcast_axis_partitions` then?

Reply:
I suppose we can do this, since it will not change the behavior of any other function, since there is no use of calling this function with the default value of `keep_partitioning` anywhere. However, what is your motivation for this change?

Reply:
Actually, I think the better value would be False for all related functions because a dataframe tends to change the shape and we want it to be split into NPartitions value. We can discuss this in a separate issue.